src/s3gw_ui_backend.py

```diff
@@ -19,15 +19,30 @@
 from typing import Awaitable, Callable
 
 import uvicorn
-from fastapi import FastAPI
+from fastapi import FastAPI, Response
 from fastapi.logger import logger
 from fastapi.staticfiles import StaticFiles
+from starlette.types import Scope
 
 from backend.api import admin, auth, buckets, config, objects
 from backend.config import Config
 from backend.logging import setup_logging
 
 
+class S3gwStaticFiles(StaticFiles):
```

Comment:
I think it would be best if we added a small comment here letting us know why we're not caching the index file